src/client-side-metrics/gcp-metrics-handler.ts

```diff
@@ -43,14 +43,151 @@ interface MetricsInstruments {
   clientBlockingLatencies: typeof Histogram;
 }
 
+/**
+ * This method gets the open telemetry instruments that will store GCP metrics
+ * for a particular project.
+ *
+ * @param projectId The project for which the instruments will be stored.
+ * @param exporter The exporter the metrics will be sent to.
+ */
+function getInstruments(projectId: string, exporter: PushMetricExporter) {
```

Comment:
It's a bit confusing that there are two `getInstruments` functions. I'd suggest calling this `createInstruments` or `initilizeInstrumentsForProject` or something

Reply:
It is now createInstruments. Done.